src/Storage/Storage.Management/StorageAccount/NewAzureStorageAccountManagementPolicyFilter.cs

```diff
@@ -0,0 +1,47 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Management.Storage.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Storage;
+using Microsoft.Azure.Management.Storage.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Management.Storage
+{
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "StorageAccountManagementPolicyFilter"), OutputType(typeof(PSManagementPolicyRuleFilter))]
+
+    public class NewAzureStorageAccountManagementPolicyFilterCommand : StorageAccountBaseCmdlet
+    {
+        private string[] blobType = new string[] { AzureBlobType.BlockBlob };
```

Comment:
Can you just put this into the ExecuteCmdlet call?

Reply:
Fxied, by add it to ExecuteCmdlet().